deploy/sre-prometheus/100-control-plan-leader-election-status.PrometheusRule.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-leader-election-master-status-alerts
+    role: alert-rules
+  name: sre-leader-election-master-status-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-leader-election-master-status-alerts
+    rules:
+    - alert: ControlPlaneLeaderElectionFailingSRE
+      expr: sum(leader_election_master_status{container=~"kube-scheduler|provisioner-kube-rbac-proxy|kube-controller-manager" }) by (container) < 1
+      for: 10m
+      labels:
+        severity: warning
+        namespace: openshift-monitoring
```

Comment:
```suggestion
        namespace: openshift-monitoring
        dependability: immature
        source: https://issues.redhat.com/browse/OSD-9114
```

Reply:
given that this alert requires some validation, maybe we should start to standardize and use labels like the above? Thoughts @sam-nguyen7 @blrm

Reply:
Discussing labels in `SREP Alerting Practices` doc

Reply:
Per https://coreos.slack.com/archives/C01M1NRKUF4/p1653911544848009 we will merge without these labels